File: fix.js
const fs = require('fs');

function processFile(path) {
  let content = fs.readFileSync(path, 'utf8');
  
  const pattern1 = /<div key=\{key\}>\s*<div className=\"text-xs font-bold text-\[?#[a-zA-Z0-9]+\]? mb-1\.5\">\s*(\{.*?\})\s*<\/div>\s*<div className=\"text-sm text-\[?#[a-zA-Z0-9]+\]? font-medium bg-white border border-\[?#[a-zA-Z0-9]+\]? p-3 rounded-xl whitespace-pre-wrap\">\s*\{value \|\| \'\-\'\}\s*<\/div>\s*<\/div>/g;
  
  const pattern2 = /<div key=\{key\}>\s*<div className=\"text-xs font-bold text-[a-zA-Z0-9]+-500 mb-1\.5\">\s*(\{.*?\})\s*<\/div>\s*<div className=\"text-sm text-[a-zA-Z0-9]+-900 font-medium bg-white border border-[a-zA-Z0-9]+-200 p-3 rounded-xl whitespace-pre-wrap\">\s*\{value \|\| \'\-\'\}\s*<\/div>\s*<\/div>/g;

  function repl(match, keyExp) {
    return `{(() => {
                                const isHighlight = /이름|고객명|연락처|입금자명|전화번호|상품명|예약 날짜|예약 시간/.test(key);
                                return (
                                  <div key={key} className="flex flex-col mb-1.5">
                                    <div className="flex items-center gap-2 mb-1">
                                      <div className="w-1 h-3.5 bg-indigo-500 rounded-full"></div>
                                      <div className="text-[14px] font-extrabold text-gray-800">{${keyExp.replace(/[{}]/g, '')}}</div>
                                    </div>
                                    <div className={\`p-3.5 rounded-xl border \${isHighlight ? 'bg-indigo-50 border-indigo-200 text-indigo-900 text-[15px] font-extrabold shadow-sm' : 'bg-gray-50/80 border-gray-100 text-gray-800 text-[14px]'} whitespace-pre-wrap\`}>
                                      {value || '-'}
                                    </div>
                                  </div>
                                );
                              })()}`;
  }
                              
  content = content.replace(pattern1, repl);
  content = content.replace(pattern2, repl);

  fs.writeFileSync(path, content, 'utf8');
}

processFile('src/app/delivery-list/page.tsx');
processFile('src/app/reservation-list/page.tsx');
processFile('src/app/order-list/page.tsx');

File: fix.py
import re

with open('src/app/page.tsx', 'r', encoding='utf-8') as f:
    c = f.read()

# The regex to find the bottom nav block inside the condition
pattern = re.compile(
    r'(<div className="w-full max-w-6xl mx-auto flex-1">[\s\S]*?</div>)\s*\{\/\* 앱 하단 네비게이션 바 \(홈 / 마이\) \*\/\}[\s\S]*?<div className="fixed bottom-0 left-0 right-0 bg-white border-t border-gray-100 flex items-center justify-around pb-safe shadow-\[0_-5px_15px_-10px_rgba\(0,0,0,0\.05\)\] z-50 h-16 sm:h-16">[\s\S]*?<\/div>\s*<\/div>\s*\)\}',
    re.MULTILINE
)

def replacer(match):
    content_area = match.group(1)
    return content_area + """
        </div>
      )}

      {/* 앱 하단 네비게이션 바 (홈 / 마이) */}
      <div className="fixed bottom-0 left-0 right-0 bg-white border-t border-gray-100 flex items-center justify-around pb-safe shadow-[0_-5px_15px_-10px_rgba(0,0,0,0.05)] z-50 h-16 sm:h-16">
        <button 
          onClick={() => setActiveTab('dashboard')}
          className={`flex flex-col items-center justify-center w-full h-full space-y-1 ${activeTab !== 'my' && activeTab !== 'account' && activeTab !== 'settings' ? 'text-indigo-600' : 'text-gray-400 hover:text-gray-600'}`}
        >
          <Home className="w-5 h-5 sm:w-6 sm:h-6" />
          <span className="text-[10px] sm:text-[11px] font-bold">홈</span>
        </button>
        <button 
          onClick={() => setActiveTab('my')}
          className={`flex flex-col items-center justify-center w-full h-full space-y-1 ${activeTab === 'my' || activeTab === 'account' || activeTab === 'settings' ? 'text-indigo-600' : 'text-gray-400 hover:text-gray-600'}`}
        >
          <User className="w-5 h-5 sm:w-6 sm:h-6" />
          <span className="text-[10px] sm:text-[11px] font-bold">마이</span>
        </button>
      </div>"""

new_c, count = pattern.subn(replacer, c)
print(f"Replaced {count} times")

with open('src/app/page.tsx', 'w', encoding='utf-8') as f:
    f.write(new_c)
